setup.py

```diff
@@ -7,6 +7,12 @@
     include_package_data=True,
     install_requires=[
         "Click",
+        "boto3",
+        "smart-open",
+        (
+            "dspace-python-client @ git+https://github.com/mitlibraries/"
+            "dspace-python-client@0.1.0#egg=dspace"
+        ),
```

Comment:
No, this part I knew. It was adding git to the image in the right place.

Reply:
Oh jk, missed which file this was in. Yes. Yes it was :)

Reply:
Is this working here? I needed to update `dspace-python-client@0.1.0#egg=dspace` to `dspace-python-client@0.1.0#egg=dspace-python-client` to install the client with pipenv

Reply:
Interestingly it is working currently because of the way we combine pipenv and setup.py in the Dockerfile (you can `make dist` and then `docker run submitter:latest` locally to confirm), but I think we should update it just to be consistent across our apps. I'll add that to the other changes I'm making.